### PR TITLE
Replace Spring's XML bean definition with Java Configuration

### PR DESCRIPTION
**Pull request by IvanBorislavovDimitrov**
LMCROSSITXSADEPLOY-2358

### CODE REVIEW

File: com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/PackageMarker.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.sap.cloud.lm.sl.cf;
+
+/**
+ * Used for Spring component scanning. See {@link com.sap.cloud.lm.sl.cf.PackageMarker}.
```
Comment (radoslav-d) [inline com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/PackageMarker.java:4]: Is this comment needed? :D

File: com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/configuration/FileStorageConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    @Bean
+    public FileSystemFileStorageFactoryBean fileSystemFileStorage() {
+        return new FileSystemFileStorageFactoryBean(FS_STORAGE_SERVICE_NAME);
+    }
+
+//    @Bean
+//
+//    public ObjectStoreFileStorageFactoryBean objectStoreFileStorage() {
+//        return new ObjectStoreFileStorageFactoryBean(OBJECT_STORE_SERVICE_NAME);
+//    }
```
Comment (radoslav-d) [inline com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/configuration/FileStorageConfiguration.java:24]: do not forget to uncomment or remove

File: com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/AuthorizationFilter.java [hunk 3]
```diff
@@ -18,6 +19,7 @@
 import com.sap.cloud.lm.sl.cf.web.message.Messages;
 import com.sap.cloud.lm.sl.cf.web.util.SecurityContextUtil;
 
+@Named("authFilter")
```
Comment (radoslav-d) [inline com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/AuthorizationFilter.java:22]: why not `@Named("authorizationFilter")`?

File: com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/CsrfAccessDeniedHandler.java [hunk 4]
```diff
@@ -11,6 +12,7 @@
 import org.springframework.security.web.csrf.InvalidCsrfTokenException;
 import org.springframework.security.web.csrf.MissingCsrfTokenException;
 
+@Named("accessDeniedHandler")
```
Comment (radoslav-d) [inline com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/CsrfAccessDeniedHandler.java:15]: same

**Review by radoslav-d — APPROVED** [review]

File: com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/PackageMarker.java (hunk 1 above)
Comment (radoslav-d) [reply com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/PackageMarker.java:4]: Maybe just the link to the class needs to be changed to the spring config class

File: com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/AuthorizationFilter.java (hunk 3 above)
Comment (IvanBorislavovDimitrov) [reply com.sap.cloud.lm.sl.cf.web/src/main/java/com/sap/cloud/lm/sl/cf/web/security/AuthorizationFilter.java]: Reused the names first but updated the name now

**Review by radoslav-d — APPROVED** [review]

**Pull request merged**